chains/connextsepolia/metadata.yaml

```
@@ -11,13 +11,13 @@ chainId: 6398
 deployer:
   name: Everclear
   url: https://everclear.org
-displayName: Connext Sepolia
+displayName: Everclear Sepolia
 domainId: 6398
 gasCurrencyCoinGeckoId: ethereum
 index:
   from: 4950
 isTestnet: true
-name: connextsepolia
+name: everclearsepolia
```

changing the name is a breaking change. we'll keep it connextsepolia for now
```suggestion
name: connextsepolia
```